backend/src/cache/client_manager.go

```diff
@@ -49,18 +52,31 @@ func (c *ClientManager) KubernetesCoreClient() client.KubernetesCoreInterface {
 	return c.k8sCoreClient
 }
 
+func (c *ClientManager) MinioClient() api_storage.ObjectStoreInterface {
+	return c.minioClient
+}
+
 func (c *ClientManager) Close() {
 	c.db.Close()
 }
 
-func (c *ClientManager) init(params WhSvrDBParameters, clientParams util.ClientParameters) {
+func (c *ClientManager) init(params WhSvrDBParameters, clientParams util.ClientParameters, s3params S3Params) {
 	timeoutDuration, _ := time.ParseDuration(DefaultConnectionTimeout)
 	db := initDBClient(params, timeoutDuration)
 
 	c.time = util.NewRealTime()
 	c.db = db
 	c.cacheStore = storage.NewExecutionCacheStore(db, c.time)
 	c.k8sCoreClient = client.CreateKubernetesCoreOrFatal(timeoutDuration, clientParams)
+	c.minioClient = NewMinioClient(timeoutDuration, s3params)
+}
+
+func NewMinioClient(initConnectionTimeout time.Duration, params S3Params) api_storage.ObjectStoreInterface {
+	glog.Infof("NewMinioClient %v", params)
+	minioClient := api_client.CreateMinioClientOrFatal(params.serviceHost, params.servicePort,
+		params.accessKey, params.secretKey, params.isServiceSecure, params.region, initConnectionTimeout)
+	return api_storage.NewMinioObjectStore(&api_storage.MinioClient{Client: minioClient},
```

Comment:
here you just use region. Maybe that is the best term for consistency

Reply:
ok, will use the same shorter form